src/main/icons/generate_icons.sh

```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env bash
 for size in 16 24 32 48 64; do
-    convert icon.png -resize ${size}x${size} base/${size}.png
+    convert icon_enabled.png -resize ${size}x${size} base/${size}.png
```

Comment:
This script is somewhat counter intuitive, but is only needed to convert the icon which will be used within various file explorers for Cadmus, rather than internally anywhere in the code, so I'd venture to say that the changes to this file aren't needed. Otherwise, this PR looks good & I'll merge it. Thanks!